lib/scrimp/app.rb

```diff
@@ -55,6 +55,19 @@ class App < Sinatra::Base
       protocols.to_json
     end
 
+    get '/transports' do
+      transports = {'Thrift::BufferedTransport'=>'Thrift::BufferedTransport',
```

Comment:
I'd like to switch the order of these two, so that FramedTransport is the default, to avoid confusing existing users who may upgrade and not notice they need to change the transport field.